website/server/models/user/methods.js

```diff
@@ -20,6 +20,49 @@ schema.methods.getGroups = function getUserGroups () {
   return userGroups;
 };
 
+// Get an array of error message keys that would be thrown if the given interaction was attempted
+schema.methods.getObjectionsToInteractionIfAny = function getObjectionsToInteractionIfAny (interaction, receiver) {
+  let sender = this;
+
+  /* eslint-disable no-unused-vars */
```

Comment:
The checks are all of the form `(sndr, rcvr) => ...`, due to being iterated over and called with the same signature, but not all of them use both parameters. This leads to spurious `no-unused-vars` issues.

I think following the linter here would be wrong and make the code worse, not better. I've added a comment stating the reason for the disable.